components/sequencer/mysql/mysql.go

```diff
@@ -0,0 +1,156 @@
+//
+// Copyright 2021 Layotto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package mysql
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"mosn.io/layotto/components/pkg/utils"
+	"mosn.io/layotto/components/sequencer"
+	"mosn.io/pkg/log"
+)
+
+type MySQLSequencer struct {
+	metadata   utils.MySQLMetadata
+	biggerThan map[string]int64
+	logger     log.ErrorLogger
+	ctx        context.Context
+	cancel     context.CancelFunc
+}
+
+func NewMySQLSequencer(logger log.ErrorLogger) *MySQLSequencer {
```

Comment:
should add import in main.go 

![image](https://user-images.githubusercontent.com/11661906/167853228-6b9c46a8-972e-4dd4-8cb6-703f76137fbe.png)